### ISSUE TITLE
Merging HTML styled text double in DOCX files with velocity

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#134]
```
It tested the new version 0.9.8 instaed of 0.9.7 before. I uses HTML 
textstyling with velocity. With changing the version the same text with appear 
double inside the DOCX file.

With Input <p>my text</p> I will get (in two lines / paragraphs):

mytext
mytext

I cannot test it with freemarker or with ODT actually. Your Demo Webapp looks 
OK in this comibantion.
```

Original issue reported on code.google.com by `googlec...@norz.de` on 20 Jul 2012 at 12:30

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Do you have the same problem with 0.9.7? Could you attach please your docx+Java 
code (Pojo, Main).

Many thank's 

Regards Angelo
```

Original comment by `angelo.z...@gmail.com` on 20 Jul 2012 at 12:44
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
No, the problem is only with 0.9.8. I switched back to the old 0.9.7 because 
I'm very busy in my project. Unfortunately I couldn't write an example sniplet 
of my code yet.

I think every time you put <p>my text</p> into the Velocity/HTMLStyle/DocX 
process you will get the doubled text. Without the paragraph tags you won't 
have this issue. Strange...

Regards Alexander
```

Original comment by `googlec...@norz.de` on 25 Jul 2012 at 10:49

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Could you attach your docx please? Without sample I cannot help you, sorry.

Regards Angelo
```

Original comment by `angelo.z...@gmail.com` on 25 Jul 2012 at 12:59

**Comment by GoogleCodeExporter** [comment]
```
Here are my example files. I will change to the new <br> tag now, that 'solves' 
also the other <p> issue 127 ;-)
```

Original comment by `googlec...@norz.de` on 16 Aug 2012 at 11:49

Attachments:
- [template.docx](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-134/comment-4/template.docx)
- [output.docx](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-134/comment-4/output.docx)
- [TestXDocReport.java](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-134/comment-4/TestXDocReport.java)

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Many thank's for your samples. I will study it as soon as I will have time.

Regards Angelo
```

Original comment by `angelo.z...@gmail.com` on 18 Aug 2012 at 1:14

**Comment by GoogleCodeExporter** [comment]
```
Problem is fixed in the 1.0.0.

I have add JUnit for 

* Freemarker => 
http://code.google.com/p/xdocreport/source/browse/integrationtests/fr.opensagres
.xdocreport.core.test/src/test/java/fr/opensagres/xdocreport/document/docx/prepr
ocessor/DocxPreprocessorTextStylingWithFreemarker.java

* Velocity => 
http://code.google.com/p/xdocreport/source/browse/integrationtests/fr.opensagres
.xdocreport.core.test/src/test/java/fr/opensagres/xdocreport/document/docx/prepr
ocessor/DocxPreprocessorTextStylingWithVelocity.java
```

Original comment by `angelo.z...@gmail.com` on 29 Aug 2012 at 12:42
- Changed state: **Fixed**
